tests/cypress/pageObject/dashboard/customizeYourStore.page.js

```diff
@@ -0,0 +1,100 @@
+const SELECTOR = {
+	CARD_LIST: '[data-variant=standard]',
+	HOME_PAGE: 'Home Page',
+	ABOUT_PAGE: 'About Page',
+	CONTACT_PAGE: 'Contact Page',
+	STORE_LAYOUT: 'Store Layout',
+	CUSTOMER_ACCOUNT_PAGE: 'Customer Account Page',
+
+	THEME_HOMEPAGE_TITLE: 'Homepage',
+	THEME_ABOUTPAGE_TITLE: 'About Us',
+	THEME_CONTACTPAGE_TITLE: 'Contact Us',
+	PLUBISH_BUTTON: 'Publish',
+	SITE_IDENTITY: 'Site Identity',
+	SITE_TITLE: 'Site Title',
+	STORE_LAYOUT_PUBLISH_BUTTON: '[name=save]',
+};
+
+class CustomizeYourStorePage {
+	cardList() {
+		return cy.get( '[data-variant=standard]' );
```

Comment:
Any reason we aren't using the selector here? 
```suggestion
		return cy.get( SELECTOR.CARD_LIST );
```

Reply:
yes